aws_pricing/hatch.toml

```
@@ -1,7 +1,7 @@
 [env.collectors.datadog-checks]
 
 [[envs.default.matrix]]
-python = ["2.7", "3.9"]
+python = ["3.9"]
```

```suggestion
python = ["3.11"]
```

We recently upgraded to 3.11, thus this will need to be updated as well.